.github/workflows/WebAutomatedTests.yml

```diff
@@ -12,26 +12,26 @@ jobs:
         runs-on: ubuntu-latest
 
         steps:
+            - name: "Checking-out code"
+              uses: actions/checkout@v2
+              with:
+                submodules: true
             - name: "Checking changed files"
               id: files
-              uses: softprops/diffset@v1
+              uses: technote-space/get-diff-action@v4
               with:
-                global_files: |
+                DIFF_FILTER: 'AMRCD'
```

Comment:
What does this mean? 😮

Reply:
Add, Modified, Renamed, Created or Deleted :P

Reply:
But i changed the whole PR 😅